packages/x-charts-pro/README.md

```diff
@@ -15,7 +15,7 @@ This component has the following peer dependencies that you will need to install
 
 ```json
 "peerDependencies": {
-  "@mui/material": "^5.15.14 || ^6.0.0",
+  "@mui/material": "^5.15.14 || ^6.0.0 || ^7.0.0",
```

Comment:
as long as core is in beta, should they contain beta as well? that would make it easier to test

Reply:
oh no, I didn't see that it was README 😄 nvm